stripe/src/main/java/com/stripe/android/Stripe.java

```diff
@@ -1,5 +1,9 @@
 package com.stripe.android;
 
+import android.util.Log;
+
+import java.util.concurrent.Executor;
```

Comment:
Ah, yes, good catch.  I was using the Log but removed it, will clean up.  I'm not sure about the Executor as I did not explicitly add that but AS may have, let me research.